### PR TITLE
Admins will get pop up asking them to start BR if the round end was delayed

### LINKED ISSUE #3060: Make battle royal automatically start if restart is delayed.
Feature requests are handled in the repository, but you could also discuss your ideas on beestation discord or forums here: https://forums.beestation13.com/

### PR DESCRIPTION
**Pull request by Archanial**
## About The Pull Request

Title.
closes: #3060 because it's as close to BR "automatically" start as we can get.
Here's how it looks:
obraz
how it looks with dark chat:
obraz

PR that disables some logging (most importantly attack logs) will be next.

## Why It's Good For The Game

I guess people like BR?

## Changelog
:cl:
admin: Admins will now get a pop up asking them if they want to start Battle Royale if the round was delayed.
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
Make the buttons not inline (or right aligned). Also why is the background purple and the bar yellow?

**Comment by Archanial** [comment]
> 
> 
> Make the buttons not inline (or right aligned). Also why is the background purple and the bar yellow?

Because I didn't touch the bar's color but I've edited background's color. I'll tweak the buttons.

File: tgui/packages/tgui-panel/styles/components/Stat.scss [hunk 1]
```diff
@@ -43,6 +43,16 @@ $border-color: color.scale(colors.fg(colors.$primary), $lightness: 75%) !default
     rgba(#e6a648, 0) 40%);
 }
 
+.StatBorder_br {
+  border-left: (1em / 6) solid #e6a648;
```
Comment (PowerfulBacon) [inline tgui/packages/tgui-panel/styles/components/Stat.scss]: Change this to 844cc3 too or something unique for be button, keep the bar the same colour as the background.

**Comment by ike709** [comment]
Needs a confirmation popup.

**Comment by Archanial** [comment]
> Needs a confirmation popup.

It launches br admin proc which has confirmation pop up

File: tgui/packages/tgui-panel/stat/reducer.js [hunk 2]
```diff
@@ -65,5 +66,18 @@ export const statReducer = (state = initialState, action) => {
       dead_popup: false,
     };
   }
+  if (type === 'stat/alertBr') {
+    return {
+      ...state,
+      alert_popup: null,
```
Comment (PowerfulBacon) [inline tgui/packages/tgui-panel/stat/reducer.js:72]: I think the loop that sends the alert message to people should just send the BR alert to admins and never send them the round end alert at all.

File: code/modules/tgui_panel/stat.dm [hunk 3]
```diff
@@ -112,3 +114,26 @@
...
+ * public
+ *
+ * Displays the message asking an admin to start battle royale
+ */
+/datum/tgui_panel/proc/give_br_popup(title, text)
+	if(!is_ready())
+		return
+	var/list/payload = list()
+	payload["title"] = title
+	payload["text"] = text
```
Comment (PowerfulBacon) [inline code/modules/tgui_panel/stat.dm]: I wouldnt be against hardcoding this, since it never changes and removes some data sent to TGUI

**Review by PowerfulBacon — DISMISSED** [review]

**Review by ike709 — APPROVED** [review]
Punted this to the headmins to see if they universally hate it or not before I merge it.

**Pull request merged**